app/models/manageiq/providers/vmware/infra_manager/vim_connect_mixin.rb

```diff
@@ -53,6 +53,11 @@ def raw_connect(options)
       validate_connection do
         vim = MiqVim.new(ip, user, pass)
         raise MiqException::Error, _("Adding ESX/ESXi Hosts is not supported") unless vim.isVirtualCenter
+
+        # If the time on the vCenter/Host is very far off from MIQ system time then
```

Comment:
We can't be host here (since the above check prevents it), so I think we can remove "Host" from the comment.